docs/generated/packages/workspace/generators/convert-to-monorepo.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "convert-to-monorepo",
+  "factory": "./src/generators/convert-to-monorepo/convert-to-monorepo",
+  "schema": {
+    "$schema": "http://json-schema.org/schema",
+    "$id": "NxWorkspaceConvertToMonorepo",
+    "cli": "nx",
+    "title": "Nx Convert to Monorepo",
+    "description": "Convert a Nx project to a monorepo with apps and libs folders.",
```

Comment:
"Convert an Nx standalone project to a monorepo"

I wouldn't even mention "apps" and "libs" tbh